statuslistener/statuslistener_test.go

```diff
@@ -138,6 +139,23 @@ func FetchDataFor(resourceType string, resourceID string, forBulkMessage bool) (
 		res = res.Find(application)
 		bulkMessage["applications"] = application.Source.Applications
 		bulkMessage["endpoints"] = application.Source.Endpoints
+
+		authentication := &m.Authentication{ResourceID: application.ID,
+			ResourceType:               "Application",
+			ApplicationAuthentications: []m.ApplicationAuthentication{},
+		}
+
+		authDao := &dao.AuthenticationDaoImpl{TenantID: &application.TenantID}
+		authenticationsByResource, err := authDao.AuthenticationsByResource(authentication)
+		if err != nil {
+			return err, nil
+		}
```

Comment:
I probably should have seen this - why are we not returning err last? If we're going to return an error type we should always return it last.

Reply:
method where is `return` even doesn't return error, I will use panic instead as it is code related to tests.